viewer-core/src/main/java/org/edmcouncil/spec/fibo/weasel/model/graph/viewer/converter/NodeConverter.java

```diff
@@ -0,0 +1,47 @@
+package org.edmcouncil.spec.fibo.weasel.model.graph.viewer.converter;
+
+import org.edmcouncil.spec.fibo.weasel.model.graph.GraphNodeType;
+
+/**
+ * @author Patrycja Miazek (patrycja.miazek@makolab.com)
+ */
+public class NodeConverter {
+
+  private int cardinality;
+  private boolean optional;
+  private GraphNodeType type;
+  private String iri;
```

Comment:
I think the class with that name cannot contain this fields. If this is a converter this cannot storage graph-node data. Delete or refactor.